### PR TITLE
Create Dockerfile

### PR DESCRIPTION
**Pull request by richb-hanover**
I like to run programs in Docker, specifically to avoid the concerns about version numbers of software that I don't use regularly.

The Dockerfile builds a container that uses Python 3.8 and Flask to run the Web Bluetooth API application. In so doing, I needed to change the port from 5000 to 5001 (I'm not sure why - but it made Docker work...) 

I updated the README to show how to build and run the Docker container. Finally, I added in MVP.css to "pretty up" the web page.

Thanks for this tool! Now I have another toy to play with!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,9 +11,34 @@ The goal of this project is to reverse engineer and open source the TrackR devic
 
 ## Instructions
 The proof-of-concept uses a Python Flask server to host a single HTML/JS page that one can use to connect to and trigger a TrackR device. To set it up, you will need to:
+
 1. Install Python 3.x
 2. Install Flask (`pip install flask`)
 3. Clone the repo
-4. Run app.py (CD to the folder containing main.py and run `python app.py`)
-5. Go to http://127.0.0.1:5000/ in a browser that supports the Web Bluetooth API (such as Chrome).
+4. Run app.py (`cd` to the folder containing app.py and run `python app.py`)
```
Comment (danielweidman) [inline README.md:18]: Good catch!

**Review by danielweidman — APPROVED** [review]
This is awesome! I love the detailed instructions to go with the dockerfile, and the CSS you added definitely makes it a lot prettier. Thank you!

**Comment by richb-hanover** [comment]
Thanks! Happy to do it.

In the spirit of full disclosure, I did not test the bare python build on port 5001... I expect it'll be fine, but perhaps you could check it... Thanks again.

**Comment by danielweidman** [comment]
The bare Python build works okay in port 5001, I tested it.

That said, my main dev flow has involved using a Flask launch configuration in PyCharm, which overrides the port specified in app.py to 5000 anyway (unless it's changed in the a Pycharm launch configuration).

**Pull request merged**